TrackletAlgorithm/InputRouterTop.cc

```diff
@@ -1,19 +1,27 @@
 #include "InputRouterTop.h"
 
+
 void InputRouterTop( const BXType bx
+  , const unsigned int hNmemories // number of memories 
```

Comment:
@sseifeln This argument doesn't seem to go anywhere. Can you remove it?

Reply:
sure - will have a look first thing Monday and commit :)

Reply:
@aehart  - I think I'm too late and you already did it (was way off on hardware assistance required for the week!)?

Reply:
It helps! This part looks good with your latest commit.